boards/arm/tiva/tm4c1294-launchpad/src/tm4c_bringup.c

```diff
@@ -151,6 +152,46 @@ static void tm4c_i2ctool(void)
 #  define tm4c_i2ctool()
 #endif
 
+/****************************************************************************
+ * Name: tm4c_bmi160_setup
+ *
+ * Description:
+ *   Initialize and register the bmi160 sensor.
+ *
+ * Input Parameters:
+ *   bus - A number identifying the I2C bus.
+ *
+ * Returned Value:
+ *   On success, zero (OK) is returned.  On failure, a negated errno value
+ *   is returned to indicate the nature of the failure.
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_SENSORS_BMI160
+static int tm4c_bmi160_setup(int bus)
+{
+  int ret;
+  struct i2c_master_s *i2c;
+
+  /* Initialize i2c device */
+
+  i2c = tiva_i2cbus_initialize(bus);
+  if (!i2c)
+    {
+      _err("ERROR: Failed to initialize i2c%d.\n", bus);
```

Comment:
Maybe you can replace _err() with syslog() too!

Reply:
Why?

Reply:
There is a combination of _err(), serr(), and syslog() in this file. Honestly I do not know whether any/some/all of these should change or stay as-is, and, if any need to change, then what they should change to... I'm okay with leaving it as-is until we know specifically what is desired.